README.md

```diff
@@ -0,0 +1,23 @@
+# SpamBlockers
+
+## Requirements
+* Node.js (v12.x or later is recommended) with `npm` (latest version recommended)
+* MongoDB server
+* Telegram Bot API token from BotFather
+* Log chat (optional)
```

Comment:
This should say "Log channel".

Reply:
@YouTwitFace Why should I log bans/unbans in a channel?

Reply:
That's how it is in the `.env` and consistency is nice.

Reply:
@YouTwitFace Every action should be audited.

Reply:
@robbyoconnor I agreed too, @robbyoconnor.

Reply:
@AndreiJirohHaliliDev2006 Transparency is the key to trust admins too.